public/scripts/exif-parser-0.1.12-min.js

```diff
@@ -0,0 +1 @@
gs.resolveTagNames?(tags={},getTagValue=function(t){return tags[t.name]},setTagValue=function(t,value){tags[t.name]=value}):(tags=[],getTagValue=function(t){var i;for(i=0;i<tags.length;++i)if(tags[i].type===t.type&&tags[i].section===t.section)return tags.value},setTagValue=function(t,value){var i;for(i=0;i<tags.length;++i)if(tags[i].type===t.type&&tags[i].section===t.section)return void(tags.value=value)}),jpeg.parseSections(stream,function(sectionType,sectionStream){var sectionOffset=sectionStream.offsetFrom(start);225===sectionType?exif.parseTags(sectionStream,function(ifdSection,tagType,value,format){if(flags.readBinaryTags||7!==format){if(513===tagType){if(thumbnailOffset=value[0],flags.hidePointers)return}else if(514===tagType){if(thumbnailLength=value[0],flags.hidePointers)return}else if(259===tagType&&(thumbnailType=value[0],flags.hidePointers))return;if(flags.returnTags)if(flags.simplifyValues&&(value=simplify.simplifyValue(value,format)),flags.resolveTagNames){var sectionTagNames=ifdSection===exif.GPSIFD?tagNames.gps:tagNames.exif,name=sectionTagNames[tagType];name||(name=tagNames.exif[tagType]),tags.hasOwnProperty(name)||(tags[name]=value)}else tags.push({section:ifdSection,type:tagType,value:value})}})&&(app1Offset=sectionOffset):flags.imageSize&&"SOF"===jpeg.getSectionName(sectionType).name&&(imageSize=jpeg.getSizeFromSOFSection(sectionStream))}),flags.simplifyValues&&(simplify.castDegreeValues(getTagValue,setTagValue),simplify.castDateValues(getTagValue,setTagValue)),new ExifResult(start,tags,imageSize,thumbnailOffset,thumbnailLength,thumbnailType,app1Offset)}},module.exports=Parser},{"./exif":7,"./exif-tags":6,"./jpeg":8,"./simplify":10}],10:[function(require,module,exports){var exif=require("./exif"),date=require("./date"),degreeTags=[{section:exif.GPSIFD,type:2,name:"GPSLatitude",refType:1,refName:"GPSLatitudeRef",posVal:"N"},{section:exif.GPSIFD,type:4,name:"GPSLongitude",refType:3,refName:"GPSLongitudeRef",posVal:"E"}],dateTags=[{section:exif.SubIFD,type:306,name:"ModifyDate"},{section:exif.SubIFD,type:36867,name:"DateTimeOriginal"},{section:exif.SubIFD,type:36868,name:"CreateDate"},{section:exif.SubIFD,type:306,name:"ModifyDate"}];module.exports={castDegreeValues:function(getTagValue,setTagValue){degreeTags.forEach(function(t){var degreeVal=getTagValue(t);if(degreeVal){var degreeRef=getTagValue({section:t.section,type:t.refType,name:t.refName}),degreeNumRef=degreeRef===t.posVal?1:-1,degree=(degreeVal[0]+degreeVal[1]/60+degreeVal[2]/3600)*degreeNumRef;setTagValue(t,degree)}})},castDateValues:function(getTagValue,setTagValue){dateTags.forEach(function(t){var dateStrVal=getTagValue(t);if(dateStrVal){var timestamp=date.parseExifDate(dateStrVal);void 0!==timestamp&&setTagValue(t,timestamp)}})},simplifyValue:function(values,format){return Array.isArray(values)&&(values=values.map(function(value){return 10===format||5===format?value[0]/value[1]:value}),1===values.length&&(values=values[0])),values}}},{"./date":4,"./exif":7}]},{},[1]);
```

Comment:
What is this for?

Reply:
we actually do want this... ,but i dont understand why it wasn't already checked in?

https://github.com/AudiusProject/audius-client/blob/e42bf86d7e6ed496e0bb1ffbd03743c75083dc40/packages/web/scripts/publishScripts.sh#L11

Reply:
i think this is a holdover from before mono-repo that ive just had unchecked for months lol, removing...

Reply:
LOL